rdtools/clearsky_temperature.py

```diff
@@ -85,11 +85,13 @@ def get_clearsky_tamb(times, latitude, longitude, window_size=40,
         day = _get_pixel_value(a, lon_index, lat_index, k, radius)
         night = _get_pixel_value(b, lon_index, lat_index, k, radius)
 
+        # Following code results in errors, likely incorrect
+        '''
```

Comment:
Should we just take this out, or is there value in preserving the history explicitly in code instead of hiding it away in the git history?

Reply:
I'm fine with removing it.